analyticConfigurations/metric_meta-consul.json

```diff
@@ -4,9 +4,8 @@
       {
         "match": "consul.catalog.(total|nodes|services)_.*",
         "properties": {
-          "tags": {
-            "unit": "count"
-          }
+          "statistic": "MAX",
+          "type": "COUNTER"
```

Comment:
This should only be set if the sample values are ever increasing. An example of that would be a count of all transactions ever processed by a node.

Reply:
I've removed the "type" definition for this metric